docs/source/running-autometa.rst

```diff
@@ -2,8 +2,28 @@
 Running Autometa
 ================
 
+
 Before running anything make sure you have activated the conda environment using
- ``conda activate autometa`` and are in the Autometa directory.
+
+``conda activate autometa`` and are in the Autometa directory.
```

Comment:
This is a nitprick and you can probably ignore it, but is it possible to add `conda activate autometa` in the same line (in the output I mean). I know this is what we tried last time and the output was pretty weird, but I have been people do that using sphinx. probably there is some way around.

Reply:
Like `NODE_235_length_161117_cov_116.709` below. I gues it's possible

Reply:
What do you mean add `conda activate autometa` in the output?

Reply:
By output I mean the html page we see the doc on

Reply:
Ah I see. You want the command to be _inline_. Sure thing. Fixed 👍